apps/storybook/src/DomainWidget.stories.tsx

```diff
@@ -12,6 +12,7 @@ const meta = {
   decorators: [DomainWidgetDecorator],
   args: {
     customDomain: undefined,
+    onCustomDomainChange: () => {},
```

Comment:
This is the only required event handler prop in the entire lib, so I get an error if it's not specified in `args`.

I'm passing a no-op function for now, but at some point it will receive an explicit mock function: `onCustomDomainChange: fn()` (cf. updated PR description).